index.js

```diff
@@ -354,6 +354,21 @@ module.exports = function () {
 
         break;
 
+      case 'h':
+
+        console.log("Keyboard shortcuts:");
```

Comment:
- Please abstract this out to another function (say `print_help`).
- Print the help only `if (!solving && !inspecting && !post_inspecting)`